exercises/es/solution_02_02_02.py

```diff
@@ -1,7 +1,7 @@
 import spacy
 
-nlp = spacy.load("en_core_web_sm")
-doc = nlp("David Bowie is a PERSON")
+nlp = spacy.load("es_core_news_sm")
+doc = nlp("David Bowie tiene el label PERSON")
```

Comment:
The Spanish label would be `PER` here, so we should probably change it to not make it confusing.